skimage/draw/__init__.py

```diff
@@ -1,3 +1,5 @@
+"""Utilities for generating coordinates of different shapes and curves.
+"""
```

Comment:
so, where do you want this long description to be written ?

Reply:
> Looks great! What do you think about using "functionality" instead of "utilities". Does that sound better to you?

@lagru I don't agree... I think that 'utilities' could perhaps be replaced by 'utility functions' or 'helper functions' (but 'utilities' is fine). 'Functionality' has a slightly different meaning, it's more abstract; the _functionality_ is what's provided by the helper function, it's the 'result' of using it... I think that "functionality for generating coordinates (...)" doesn't really work, because the generation of coordinates is the functionality itself.

Reply:
should I do something now ?

Reply:
I find @mkcor arguments, as usually, very convincing. I think I mainly suggested it because our submodules are all starting to look like "Utilities for ..." which I guess prompted me to want to mix it up. :smile: Just leave it like it is if you agree with @mkcor.

> so, where do you want this long description to be written ?

Just below the first line (leaving a blank line between). In https://github.com/scikit-image/scikit-image/issues/6761 I linked a style guide to writing docstrings. It is the same pattern that is used in a lot of scikit-images functions: Short description in imperative mood, followed by a more detailed description of the purpose and use.

Reply:
I referred numpy github repository files for detailed docstrings, let me know if this will do @lagru

Reply:
@lagru  detailed description docstring has been added to skimage.draw module, and the corresponding pull request is at #6804

Reply:
yes, #6804 supersedes this PR, so I am going to close this PR